urbanopt/geojson-modelica-translator
# feat: time series heating indirect

#### Any background context you want to provide?
Part of GMT flexibility effort
#### What does this PR accomplish?
- adds heating indirect ETS, and related couplings for use with TimeSeries load
- add graph to the templating context
- fix cooling indirect by decoding the load id
#### How should this be manually tested?
- run test_time_series_heating_indirect.py, inspect the DistrictEnergySystem.mo output, load into dymola and verify it works as expected
#### What are the relevant tickets?
#### Screenshots (if appropriate)
